common/json_stream.c

```diff
@@ -626,6 +626,14 @@ void json_add_amount_msat(struct json_stream *result,
 	json_add_u64(result, msatfieldname, msat.millisatoshis); /* Raw: low-level helper */
 }
 
+void json_add_amount_sat(struct json_stream *result,
+			  const char *satfieldname,
+			  struct amount_sat sat)
+{
+	assert(strends(satfieldname, "_sat") || streq(satfieldname, "sat"));
+	json_add_u64(result, satfieldname, sat.satoshis); /* Raw: low-level helper */
```

Comment:
Disagree.  There's a reason this function was removed.  In general, we want to expose values in msat.  In this case, the fact that the capacity has to be in whole sats is kind of a detail.